docs/endpoints/users/settings.md

```diff
@@ -1,3 +1,52 @@
 # /\_settings
 
+## Overview
+
+The `/_settings` endpoint allows users to update their profile settings and change their password. It is designed to offer users control over various aspects of their account settings through a single interface.
+
 ## POST
+
+The `POST` method is used to update user settings and change passwords.
+
+### Global Response Codes
+
+These are common HTTP status codes applicable to all requests made to this endpoint:
+
+- `200` OK: Returns the updated user object.
+- `400` Bad Request: The server cannot process the request due to a client error (e.g., malformed request syntax).
+- `401` Unauthorized: Authentication is required and has failed or has not been provided.
+- `429` Too Many Requests: The user has sent too many requests in a given amount of time ("rate limiting").
+- `500` Internal Server Error: The server has encountered a situation it doesn't know how to handle. If this error occurs, please create an issue on [GitHub](https://github.com/discuitnet/discuit).
```

Comment:
This could probably be a table.

Reply:
Also, I like that you've detailed the errors here. I might get rid of the errors section and continue this style for the other pages too. It makes more sense to do it this way, and it's easier for developers to get the information they need quickly, that's specific to that endpoint.